integration-tests/src/main/java/io/druid/testing/DockerConfigProvider.java

```diff
@@ -84,12 +84,25 @@ public String getZookeeperHosts()
         return dockerIp + ":2181";
       }
 
+      @Override
+      public String getZookeeperIntenalHosts()
```

Comment:
Intenal -> Internal

Reply:
Thanks, I fixed the typo and rebased to the latest master